src/wallet/rpc/spend.cpp

```diff
@@ -532,17 +543,14 @@ static std::vector<RPCArg> FundTxDoc(bool solving_data = true)
     return args;
 }
 
-CreatedTransactionResult FundTransaction(CWallet& wallet, const CMutableTransaction& tx, const UniValue& options, CCoinControl& coinControl, bool override_min_fee)
+CreatedTransactionResult FundTransaction(CWallet& wallet, const CMutableTransaction& tx, const std::vector<CRecipient>& recipients, const UniValue& options, CCoinControl& coinControl, bool override_min_fee)
```

Comment:
Same comment as for other `FundTransaction`. I think we should get rid of ambiguity in input parameters and don't pass `tx`. In this case it's even more confusing, because `tx.vout` is actually still checked.

Reply:
Also a good catch, we should be using the `recipients` vector to get the size here.